.github/scripts/generate_ci_workflows.py

```diff
@@ -299,6 +286,12 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
         test_runner_type=LINUX_CUDA_TEST_RUNNER,
         num_test_shards=2,
         is_scheduled="45 0,4,8,12,16,20 * * *",
+        on_pull_request=True,
+        ciflow_config=CIFlowConfig(
+            enabled=True,
+            trigger_action_only=True,
+            labels={'ciflow/scheduled'}
```

Comment:
One thing I think it's clearer is that we can review what labels a workflow is subscribing to